### DISCUSSION TITLE
Were there any breaking changes from 1.30.1 to 1.30.2?

### DISCUSSION THREAD

**Discussion by Lost-Ones** [dani-garcia/vaultwarden#4291] [Q&A]
Greetings, 

I had just performed a docker compose pull which resulted in sever performance issues and and 502 errors when trying to access the web url and the admin pages.   My previous version was 1.30.1 and pulled the new code via 'docker compose pull'   image: vaultwarden/server:latest

Question being, were there any changes that may have resulted in my issue?   I did not see anything that stood out in the release notes from a few hours ago. 

My instance of Vaultwarden is behind a caddy reverse proxy.  Time to time, it seems to respond.  If I restart the container, it seems to respond, but then quickly returns to slow performance and 502 errors.

If there is a way that I can revert to the previous code that would be helpful, and if I can provide log that is more of a debugger, let me know ( I am a bit of a docker noob still )

Regards,

**Comment by flowent59** [comment]
Hi,

Sorry for my english.

I have the same issue. in log i see that:

```
/--------------------------------------------------------------------\
|                        Starting Vaultwarden                        |
|                           Version 1.30.2                           |
|--------------------------------------------------------------------|
| This is an *unofficial* Bitwarden implementation, DO NOT use the   |
| official channels to report bugs/features, regardless of client.   |
| Send usage/configuration questions or feature requests to:         |
|   https://github.com/dani-garcia/vaultwarden/discussions or        |
|   https://vaultwarden.discourse.group/                             |
| Report suspected bugs/issues in the software itself at:            |
|   https://github.com/dani-garcia/vaultwarden/issues/new            |
\--------------------------------------------------------------------/

[INFO] Using saved config from `data/config.json` for configuration.

Error loading config:
  `ORG_ATTACHMENT_LIMIT` is out of bounds
```
In file 'data/config.json' the 'ORG_ATTACHEMENT_LIMIT' value is '-1' for me. i change the value to the same that 'USER_ATTACHEMENT_LIMIT' and it's now OK.

You must stop the container before and start after the change.

Florent

**Reply by BlackDex** [reply]
`-1` is an invalid value so that is correct.
It's nowhere documented that that is a valid value.

**Reply by flowent59** [reply]
I agree, but I don't use organizations and therefore I haven't changed any settings regarding them. I don't know where the '-1' comes from, but I know I didn't put it.

**Reply by BlackDex** [reply]
You must have configured that your self. That setting has no default value other then None, in the sense that it is optional.
So having that set to `-1` doesn't come out of nowhere.

**Comment by BlackDex** [comment]
No, there were no breaking changes. Only some extra verifications. If those break then you probably had your system configured incorrectly already.

**Reply by Lost-Ones** [reply]
Thank you for confirming.  I appreciate it.  I will check though the .env and see if I can identify any issues.

**Reply by BlackDex** [reply]
The logs should be reporting something. So that would be the best starting point.
